packages/esm-patient-flags-app/src/flags/flags-risk-count-extension/extension-config-schema.ts

```diff
@@ -0,0 +1,15 @@
+import { Type } from '@openmrs/esm-framework';
+
+export const riskCountExtensionConfigSchema = {
+  hideOnPages: {
+    _type: Type.Array,
+    _default: ['Patient Summary'],
+    _elements: {
+      _type: Type.String,
+    },
```

Comment:
```suggestion
    },
    _description: 'List of page names where the risk count badge should not be displayed. 
  Matches against the last segment of the URL path.',
```

Is this missing a _description property?